### DISCUSSION TITLE
Eliminate white space after a divider card?

### DISCUSSION THREAD

**Discussion by VitaminTe** [Clooos/Bubble-Card#641] [Questions about config, custom styles and templates] [answered]
Hello - I'm using the bubble card divider cards on my sections dashboard, I really like the look of them! That said, there is a weird amount of white space following them. Screen shot below.

<img width="390" alt="Screenshot 2024-07-08 183906" src="https://github.com/Clooos/Bubble-Card/assets/97351583/39ffe4b0-df74-4ce6-9edc-7fa2a8d842ef">

I found that if I uncheck --row-height in dev view they display perfectly without any other adverse effect. I've seen a post on the HA forms with others having the same issue. Wondering if this is something that can be fixed within bubble card or does it need to be addressed within HA? See below for the "--row-height" css I'm talking about. Unfortunately I know just enough to follow along but have no idea what I'm doing most of the time! LOL

![image](https://github.com/Clooos/Bubble-Card/assets/97351583/309c62e5-2fdf-4dbf-a6cc-c0cf9c19dade)

**Comment by Clooos** [comment, marked as answer]
That's really interesting, this should indeed allow me to adapt it for the section view.

In the meantime you can also change the card layout to "Large" in the editor, so that it will be correctly centered.

**Reply by VitaminTe** [reply]
Wonderful! The "Large" option fixes the alignment, which is a huge improvement. Eliminating the white space would make it perfect. Thanks so much for looking into it... a few more beers coming your way! :)

**Reply by Clooos** [reply]
Glad to help! And thank you in advance!
